test/scripts/update-kindnode-version.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 Authors of spidernet-io
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
should add more comment in the top to describe the usage and function

Reply:
done